services/management/deps.go

```diff
@@ -41,3 +41,7 @@ type agentsRegistry interface {
 type prometheusService interface {
 	RequestConfigurationUpdate()
 }
+
+type checksService interface {
```

Comment:
This interface is subtly different from others – we don't generate a mock for it. Should we?

/cc @BupycHuk @askomorokhov I'm having a déjà vu there – did we discuss it just recently?

Reply:
Carlos has reminded me it was discussed here https://github.com/percona/pmm-managed/pull/360

Reply:
So, I think there is no need to generate mocks until they used somewhere, but I can add them if it's common rule for the project. 
That interface named `checksService` because it hides `checks.Service`, so it's a direct inheritance of naming. However, maybe we should rename the `checks` package to `stt` or something else?

Reply:
I think we shouldn't generate until we require it. Another question: Why don't we use mocks, because we don't have tests :)
About renaming, I agree that it should be renamed to something like `stt`

Reply:
"STT" is a product / feature name. We have other features in our roadmap with different names that will use this code.